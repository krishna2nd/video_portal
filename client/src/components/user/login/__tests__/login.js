import React from 'react';
import Login from '../';
import renderer from 'react-test-renderer';

test('Login component snapshot check', () => {
  const component = renderer.create(
    <Login></Login>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});