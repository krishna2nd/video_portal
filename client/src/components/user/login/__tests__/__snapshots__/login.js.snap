// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component snapshot check 1`] = `
<div>
  <form
    className="login-user"
    onSubmit={[Function]}
  >
    <input
      className="input "
      defaultValue="ali"
      id="username"
      name="username"
      placeholder="User name"
      type="text"
    />
    <input
      className="input "
      defaultValue="password"
      id="password"
      name="password"
      placeholder="Password"
      type="password"
    />
    <button
      className="button"
      name="login"
    >
      Login
    </button>
  </form>
</div>
`;
